### PR TITLE
feat(novui): JsonSchemaForm styling

### PR DESCRIPTION
**Pull request by antonjoel82**
### What changed? Why was the change needed?

- Style JsonSchemaForm
- Add some other small styling tweaks

**Note**: _Designs do not provide comprehensive examples, nor use typical strategies for spacing in some places, so this is a best attempt. We should pass this by Nikolay for further review._

### Screenshots

Screenshot 2024-06-19 at 4 14 50 PM

Screenshot 2024-06-19 at 4 09 30 PM

### CODE REVIEW

File: libs/novui/src/tokens/semanticColors.tokens.ts [hunk 1]
```diff
@@ -189,21 +189,21 @@ export const LEGACY_COLOR_SEMANTIC_TOKENS = defineSemanticTokens.colors({
   mode: {
     cloud: {
       start: {
-        value: { base: '{colors.legacy.gradientStart}', _dark: '{colors.legacy.gradientStart}' },
+        value: { base: '{colors.legacy.gradientEnd}', _dark: '{colors.legacy.gradientEnd}' },
         type: 'color',
       },
       end: {
-        value: { base: '{colors.legacy.gradientEnd}', _dark: '{colors.legacy.gradientEnd}' },
+        value: { base: '{colors.legacy.gradientStart}', _dark: '{colors.legacy.gradientStart}' },
```
Comment (antonjoel82) [inline libs/novui/src/tokens/semanticColors.tokens.ts:196]: 🗒 note (non-blocking): These are flipped this way in our primary gradient 🙃 ‏

**Review by scopsy — APPROVED** [review]
Fantastic Joel! Thank you. Will be playing with more realistic customer usecases to see how design fits. And will collaborate with Nik in case we need to support some usecases

**Review by ainouzgali — APPROVED** [review]
Amazing @antonjoel82, it looks great!!

**Review by SokratisVidros — APPROVED** [review]
💯

File: libs/novui/src/json-schema-components/JsonSchemaForm.tsx [hunk 2]
```diff
@@ -1,68 +1,72 @@
-import React from 'react';
-import { RJSFSchema } from '@rjsf/utils';
+import Form, { FormProps } from '@rjsf/core';
```
Comment (SokratisVidros) [inline libs/novui/src/json-schema-components/JsonSchemaForm.tsx:1]: I'd argue that the JSONSchemaForm should be part of our UI library, as it is very specific to the studio part of the web app but we can debate on that later.

Comment (antonjoel82) [reply libs/novui/src/json-schema-components/JsonSchemaForm.tsx:1]: Sounds good!

**Pull request merged**